### ISSUE TITLE
Support for Python 3.5

### ISSUE THREAD

**Issue by dennissiemensma** [dennissiemensma/dsmr-reader#55]
Travis builds for Python 3.5 showed in #38 that it crashes. At the moment that's okay as Python 3.4 is the default for Raspbian. However, it should be supported in the (not so far) future.

**Comment by dennissiemensma** [comment]
Python 3.5 was 'just' released six months ago, so I'll push the milestone for it. No need to rush.

**Comment by dennissiemensma** [comment]
I recently upgraded my computer to Ubuntu 16.04 LTS, which comes with Python 3.5.2 by default. All tests seem to run well locally, so I'll adjust the Travis testing configuration to include Python 3.5 as well.

**Comment by dennissiemensma** [comment]
Confirmed, Travis seems to handle the tests in Python 3.5 well now: https://travis-ci.org/dennissiemensma/dsmr-reader/builds/173997419

For some reason Travis crashed in February earlier this year. It might also be cause by some bugfixes in either Travis or Python 3.5. 

Anyway it's good news to see that this project can be run on Python 3.5 as well. And since I did no changes to make this happen, there is a chance that earlier `dsmrreader`-releases work on Python 3.5 as well.
